specs/StoragePickers/FileSavePicker.md

```diff
@@ -21,7 +21,7 @@ runtimeclass FileSavePicker
     string SettingsIdentifier;
     string DefaultFileExtension;
     string SuggestedFileName;
-    Windows.Storage.StorageFile SuggestedSaveFile;
+    string SuggestedSaveFilePath;
```

Comment:
`TrySetSuggestedSaveFilePath` is not listed here, and the property is now read-only